src/v2/Apps/Consign/consignRoutes.tsx

```diff
@@ -18,4 +18,21 @@ export const consignRoutes = [
       }
     `,
   },
+  {
+    path: "/consign/:submissionID/offer/:offerID",
+    getComponent: () => OfferDetailApp,
+    prepare: () => {
+      OfferDetailApp.preload()
+    },
+    query: graphql`
+      query consignRoutes_OfferDetailQuery($submissionID: ID!, $offerID: ID!) {
+        submission(id: $submissionID) {
+          ...OfferDetailApp_submission
+        }
+        offer(id: $offerID, gravityPartnerId: "ssshhhhh") {
```

Comment:
`gravityPartnerId: "ssshhhhh"`: Right now, metaphysics requires a `gravityPartnerID` argument on the offer, because it's only used in CMS for partners to view their offers. @sweir27 's work in [convection#846](https://github.com/artsy/convection/pull/846) will remove the need for this argument, so when it propagates up to here, I'll just remove that argument.